### ISSUE TITLE
FigmaSharpApp UX improvements

### ISSUE THREAD

**Issue by hbons** [netonjm/FigmaSharp#55]
The FigmaSharpApp takes a Figma document URL and previews how it would be rendered using FigmaSharp. 

To do:
- [x] Page switcher
- [x] Native controls preview
- [x] Center Open dialog
- [x] Larger preview window
- [x] Center document in window (maybe add large minimum padding?) 
- [x] Show version history (depends on #49)
- [x] Fix progress indicators
- [x] Fix new window position
- [ ] Fix version/page switcher

**Comment by netonjm** [comment]
Page switcher done ;-)

**Comment by hbons** [comment]
Making a new issue for the last item.
